rtlib/rna.hh

```diff
@@ -191,6 +191,20 @@ inline void append_deep_rna(
     str.append(static_cast<char>(base_to_char(*i)));
 }
 
+// same as append_deep_rna, but LOC will leave a <X> instead of an empty string
```

Comment:
Is <X> the best way to represent this in tikz? Or is it a requirement? Would <EMPTY> not be better for understanding?

Reply:
also a short comment to `append_deep_rna` about what that does might be useful

Reply:
could be interesting for the user to know, where exactly LOC is located, e.g. for dangling bases in RNA problems in algebra functions like [edlr](https://github.com/jlab/fold-grammars/blob/26a959892938f4d42cca8fe73cec35009b7b9046/Grammars/gra_microstate.gap#L15)